### PR TITLE
The NewYork Times Page

### PR DESCRIPTION
**Pull request by Varun2323**
Description:
 The New York Times page, it shows a Subscribe Now and Log In button on the top-right corner of Navbar, a Main section related to news and a Footer section.

Header (Subscribe section):
- A link to another URL (if user wants to subscribe  or Log In).
- A button for Subscribe Now and Log In.

Main section:
- Embedded images , data and  different URLs for further information.
- A Grid applied with background image to make information more attractive.
- A concept of Flex container with Grid container inside.

Footer section:
- Defining about the different sections of news.
- Text related to legal information about the website.

### CODE REVIEW

File: index.html
Comment (ghost) [inline index.html]: Remove this comment because it doesn't highlight anything about your times section.

**Review by ghost — CHANGES_REQUESTED** [review]
# Status: Changes Required.

## Hello team,

**Great job on your project. Your attention to details is quite remarkable.**

**Please make the following changes.**

### Mandatory

1). **Remove the page horizontal scroll.**

2). **The 'space & cosmos' text, on the navigation bar, is displaying on two lines instead of one line.**

original | clone
---------| -------
newyork_original | newyork_clone

3). **The navigation bar should stick at the top of the page.**

4). **Style the image caption for the image below. As it is, your image looks like it doesn't have a caption. And the caption looks like it is part of the long middle paragraph.**

original | clone
---------| -------
bicep_original | bicep_clone

5). **The image and side text, in the editor's pick section( see screenshot below), should be links.**

editor_clone

6). **A few things to change in the screenshotted section below.**

space_clone

- **The images and image captions should be links.**

- **Add the small text, directly below the images, and the dates below the image captions like in the original (see screenshot below).**
space_original

- **The content in the most popular section is overlapping each other.  Also, each line of content should be a link.** 

7). **Wrap all the content in the main section inside a `<main>` tag and all the content in the footer section inside a `footer` tag.**

8). **The new york times logo and all the list items on the top and bottom footer section should be links.**

9). **Please add the sticky menu below at the bottom of your page(screenshot from original page).**
bottom_menu

### Happy coding 😃.

**PS: You can reach me on slack (Rose Wanjohi).**

**Review by teekaytech — CHANGES_REQUESTED** [review]
## Status: Changes Required :warning: :vertical_traffic_light: 

### Hello team :wave:

__I must say, you've done an amazing job so far. However, there are some changes that are required to be fixed before this project can be approved. Please check my points below and fix them as appropriate.__

**[ REQUIRED CHANGES ]**

- [ ] Two of your linter checks failed. Kindly fix all linter errors and ensure that the linter checks are all green.


- [ ] The previous reviewer mentioned that you should link the images with their titles In your `Editor's Pick` section. This has not been fixed. You should let the anchor tag of each side text covers their individual images.

**Also in the bottom part of the page the previous reviewer mentioned that:**
- [ ] Each of the articles in the `More in Space and Astronomy` and the `Editors' Picks` sections should be linked. 
- This includes the image and title of each of the articles. You should make them clickable.



- The same thing is applicable to the `Most Popular` section. Each of the titles should be linked.



- [ ] The fixed footer is missing in your project. Kindly add it and make it fixed to the bottom of your page.






- [ ] You are having some layout issues in the `The Theory of Inflation` section. Kindly compare them and fix them as appropriate.

Original page:


Clone page:


- Looking at the images above, it is clear that you need to work on your layout of the section mentioned.


**Other things to take care of in your footer section:**

- [ ] The New York Times logo in the footer section is linked. Kindly make it a link.

Original page:


Clone page;


- [ ] Each of the footer items is also linked. Kindly make them links.




**Please make the required changes and submit another review request when you are done.**

* * *

**_You can reach out to me on twitter (@ola_lere), Github(@teekaytech) or slack (@taofeekolalere )_**
__Happy Coding :smiley:__

**Review by jubaan — CHANGES_REQUESTED** [review]
# Status: Changes Required :recycle:

## :wave: :wave: Hello!! Great work on the project!

:100: sticky and footer navbar
:100: the website looks great

Required changes:
### [Mandatory] :red_circle: 

- [ ] the NYT logo should be a link

- [ ] eliminate ```cursors: pointer``` from css and convert link elements to anchor ```<a>``` HTML tags

- [ ] The preview screenshot has a leyend please remove it
image

- [ ] ```footer-bottom``` elements should be links
image

- [ ] social elements should be links
image
image

- [ ] wrapped what does not form part of the navbar and the footer in a ```<main>``` container, as suggested in previous review.

- [ ] missing element as mention in previous review
image

## Keep up the great work :muscle: and Happy Coding :smiley: :computer:!!

**Review by acushlakoncept — CHANGES_REQUESTED** [review]
# Status: Changes Required ♻️

#### Hello @Varun2323, Great job! 👏  
You have done well on this project. Well done 👍 
Let's make a few changes, and we will be done.

- [ ] Fix the HTML validation warnings/errors, use the [w3 Validator](https://validator.w3.org/)

- [ ] Fix the linter errors

- [ ] In this project we have to make sure nothing is messed up, aligned the text properly to the left of the image. 
Screen Shot 2020-10-06 at 10 01 24 PM

- [ ] Your '**theory of inflation**' section is not correctly implemented. Each of the grids is supposed to have their own background image, not a single image for the entire area, [see how sprite images work](https://www.w3schools.com/css/css_image_sprites.asp) for more details.
Screen Shot 2020-10-06 at 9 59 14 PM


Make these changes and submit another code review request 🙏 

### Cheers and Happy coding! 💻 👏

(Feel free to contact me on slack(@acushla), if you have any additional question)
Uduak Essien
[Twitter](https://twitter.com/acushlakoncept) | [LinkedIn](https://www.linkedin.com/in/acushlakoncept/)

**Review by jubaan — CHANGES_REQUESTED** [review]
# Status: Changes Required :recycle:

## :wave: :wave: Great job working on the required changes!!

There are still some issues related to previously required changes.

### [Mandatory] :red_circle: 

- [ ] Use semantic HTML for ```header```, ```main```, ```article```, and ```footer```. Example:
**Actual**: ``` <section class='header> ... </section>```
**Improved** :+1: : ``` <header class='header> ... </header>```

## Keep up the great work :muscle: and Happy Coding :smiley: :computer:!!

**Review by jubaan — APPROVED** [review]
# Status: Approved :tada: :tada: :tada:

You may now merge to master.

Please before submitting fix the alignment and indentation of the HTML file.

## Great work! :clap: :clap:, See you at the top :rocket: !!!

**Pull request merged**
